packages/react-native-gesture-handler/src/web/handlers/GestureHandler.ts

```diff
@@ -416,23 +416,41 @@ export default abstract class GestureHandler implements IGestureHandler {
     if (!this.viewRef) {
       throw new Error(tagMessage('Cannot handle event when target is null'));
     }
-    return {
-      nativeEvent: {
-        state: newState,
-        handlerTag: this.handlerTag,
-        oldState: oldState,
-        handlerData: {
-          pointerType: this.pointerType,
+    if (this.actionType === ActionType.NATIVE_DETECTOR) {
+      return {
+        nativeEvent: {
+          state: newState,
+          handlerTag: this.handlerTag,
+          oldState: oldState,
+          handlerData: {
+            pointerType: this.pointerType,
+            numberOfPointers: this.tracker.trackedPointersCount,
+            pointerInside: this.delegate.isPointerInBounds(
+              this.tracker.getAbsoluteCoordsAverage()
+            ),
+            ...this.transformNativeEvent(),
+            target: this.viewRef,
+          },
+        } as StateChangeEvent<unknown> | UpdateEvent<unknown>,
+        timeStamp: Date.now(),
+      };
+    } else {
```

Comment:
You can either use ternary operator for conditional return `?:`, or remove this `else` statement as it doesn't do anything 😅